src/styles/utilities/textInputAlignSelf/index.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    alignSelf: 'center',
+};
```

Comment:
Could you update this so that it's similar to the other styles in utilities like `whiteSpace` or `wordBreak` by wrapping it in an object like

```
export default {
    center: {
        alignSelf: 'center',
    },
};
```